vignettes/archive.Rmd

```diff
@@ -87,7 +87,7 @@ below). There can only be a single row per unique combination of key variables,
 and therefore the key variables are critical for figuring out how to generate a
 snapshot of data from the archive, as of a given version (also described below).
    
-```{r}
+```{r, error=TRUE}
 key(x$DT)
```

Comment:
I don't there's an error here.

Reply:
I tried for a while to reproduce the GitHub check error if this is excluded and could not.  Maybe it was a transient thing?  (Or maybe I'm trying the wrong steps to reproduce.)

Reply:
Ok. I'll leave it for now, open an issue, and mention that it's intermittent. For me, it happened on the R Studio R-CMD-Check.

Reply:
Okay, I misinterpreted; I thought it was in the CI check.  Will follow up in #81.